src/abstractdataframe/abstractdataframe.jl

```diff
@@ -334,16 +334,20 @@ end
 Report descriptive statistics for a data frame
 
 ```julia
-describe(df::AbstractDataFrame; stats = [:mean, :min, :median, :max, :nmissing, :nunique, :eltype])
+describe(df::AbstractDataFrame)
+describe(df::AbstractDataFrame, stats::Union{Symbol, Pair{Symbol}}...)
 ```
 
 **Arguments**
 
-* `df` : the AbstractDataFrame
-* `stats::Union{Symbol,AbstractVector{Symbol}}` : the summary statistics to report. If
-  a vector, allowed fields are `:mean`, `:std`, `:min`, `:q25`, `:median`,
-  `:q75`, `:max`, `:eltype`, `:nunique`, `:first`, `:last`, and `:nmissing`. If set to
-  `:all`, all summary statistics are reported.
+* `df` : the `AbstractDataFrame`
+* stats::Union{Symbol, Pair{Symbol}}... : the summary statistics to report. 
+    * Arguments can be symbols from the following: `:mean`, `:std`, `:min`, `:q25`, 
+      `:median`, `:q75`, `:max`, `:eltype`, `:nunique`, `:first`, `:last`, and 
+      `:nmissing`. The default statistics used when no `Symbol`s or `Pair`s are provided 
+      are `:mean`, `:min`, `:median`, `:max`, `:nunique`, `:nmissing`, and `:eltype`.
+    * Alternatively, specify `:all` as the only `Symbol` argument to return all statistics. 
+    * Finally, users can provide their own functions in the form of a `Pair{Symbol, Any}`.
```

Comment:
`Pair{Symbol, Any}`should be `Pair{Symbol, <:Any}`. But better use something more readable like ``"`name => function` pairs, with `name` a symbol"``.

Reply:
fixed.